Library/include/CSP/Systems/ECommerce/ECommerceSystem.h

```diff
@@ -67,14 +67,19 @@ class CSP_API ECommerceSystem : public SystemBase
 	/// @param SpaceId csp::common::String : space id of product
 	/// @param ProductId csp::common::String : Product id of product
 	/// @param Callback NullResultCallback : callback when asynchronous task finishes
-
 	CSP_ASYNC_RESULT void GetProductInformation(const common::String& SpaceId, const common::String& ProductId, ProductInfoResultCallback Callback);
 
+    /// @brief Get product information from a shopify store within a space, by providing a VariantId
+	/// @param SpaceId csp::common::String : space id of product
+	/// @param VariantId csp::common::String : Variant id of product
+	/// @param Callback NullResultCallback : callback when asynchronous task finishes
+	CSP_ASYNC_RESULT void
+		GetProductInformationByVariantId(const common::String& SpaceId, const common::String& VariantId, ProductInfoCollectionResultCallback Callback);
```

Comment:
If i understand this correctly, the api takes in a single string, and returns an array of products. This seems odd as the expectation would be for the result to be a single product. Is it possible to either return one product or to accept variantID array to let us do bulk queries ?

In contrast, the CHS work seems to take in an array id variant IDs and returns an array of products. 
CHS api example :`Task<LogicResult<VariantsQueryRoot>> FindProductsByVariantIds(string spaceId, IList<string> productVariantIds);`

Reply:
cc @MAG-DanielMurdolo for musings

Reply:
`GetProductInfoCollectionByVariantId` to align with the callback signature perhaps?

Reply:
`GetProductInfoCollectionByVariantId(s)`?